src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    @Override
+    public String toString() {
+        return description;
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public void markAsNotDone() {
+        isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "☑" : "☐"); // mark done task with tick
+    }
+
+    public static String addIndentation() {
+        return "    "; // adds an indentation of 4 spaces
+    }
```

Comment:
What is this supposed to be adding indentation to? I think reflecting it in the name, or in the comments, will make the code cleaner.

Reply:
Will consider adding additional comments to make the method clearer, thank you.